src/creature.js

```diff
@@ -146,6 +146,26 @@ export class Creature {
 			 */
 			movement: this.baseStats.movement,
 		};
+
+		this.status = {
```

Comment:
Moved these boolean statuses out of `creature.stats`. The reason is because `creature.stats` is replaced with `creature.baseStats` everytime `creature.updateAlterations()` is called, meaning any boolean statuses are unset unless applied via effects.

Applying via effects is a potential solution, but I didn't want to change too many abilities (anything applying frozen, dizzy, etc) at once.

Reply:
I see, that makes sense indeed. I always appreciate you going the extra mile and explaining things (your mindset), as I get to understand the stuff better and even improve at coding a little :-) Will probably open up an issue about this pretty soon then.